lib/app.rb

```diff
@@ -11,4 +11,15 @@ class Chitter < Sinatra::Base
     @feed_items = ChitterFeed.content
     erb :feed
   end
+
+  get '/chitter_feed/new' do
+    erb :new
+  end
+
+  post '/chitter_feed' do
+  content = params['content']
+  connection = PG.connect(dbname: 'chitter_test')
+  connection.exec("INSERT INTO feed (content) VALUES('#{content}')")
```

Comment:
these could be in another file to keep it thin

Reply:
Yeah that would be a good idea, will make sure I approach it like that in future projects 👍